app/view/mini_statement.py

```diff
@@ -0,0 +1,36 @@
+from app.model.account_transaction_details import AccountTransactionDetails
+from flask_restful import Resource
+from app.common.ResponseGenerator import ResponseGenerator
+from flask_api import status
+from app.common.logging import *
+
+
+class MiniStatement(Resource):
+
+    def get(self):
```

Comment:
This should accept account number/id as input and mini statement for that account will be returned

Reply:
Now its accepting the bank account id as input and displaying the transactions